app/templates/components/popover-message.hbs

```diff
@@ -0,0 +1,10 @@
+<i
```

Comment:
How does this look to a blind user? Can someone using a screenreader access this information?

In general we try to avoid things that *only* convey information in icon form. See http://fontawesome.io/accessibility/ for some suggestions on how to improve this. If you'd like to try this in a screen reader I'm happy to lend a hand.